components/cards/resource/resource-2/resource-2.hbs

```diff
@@ -0,0 +1,17 @@
+{{#each resource-2-cards as |card|}}
+<div class="section g--4 g--4--sl m-t500 center-mnl cd-r">
+  <div class="cdr-description">{{card.description}}</div>
+  <div class="cd-c" id="align-center"><a href="{{card.href}}"><img src="{{card.image}}" class="cdp-i p-a100" style="margin:auto; margin-bottom: 20px;"></a>
```

Comment:
Inline styles? Create modified template/css class to handle these changes if they're not background images.